WordPress/Classes/ViewRelated/Blog/JetpackSettingsViewController.swift

```diff
@@ -0,0 +1,470 @@
+                handler: { [weak self] in
+                    self?.pressedSendPushNotifications($0)
+                })
+        ]
+
+        return Section(rows: rows)
+    }
+
+    fileprivate var bruteForceAttackSection: Section {
+        let headerText = NSLocalizedString("Brute Force Attack Protection", comment: "Jetpack Settings: Brute Force Attack Protection Section")
+        let rows = [
+            Row(style: .Switch,
+                title: NSLocalizedString("Block malicious login attempts", comment: "Jetpack Settings: Block malicious login attempts"),
+                enabled: true,
+                boolValue: self.settings.jetpackBlockMaliciousLoginAttempts,
+                handler: { [weak self] in
+                    self?.pressedBlockMaliciousLoginAttempts($0)
+                }),
+
+            Row(style: .Value1,
+                title: NSLocalizedString("Whitelisted IP addresses", comment: "Jetpack Settings: Whitelisted IP addresses"),
+                enabled: self.settings.jetpackBlockMaliciousLoginAttempts,
+                handler: self.pressedWhitelistedIPAddresses)
+        ]
+
+        return Section(headerText: headerText, rows: rows)
+    }
+
+    fileprivate var wordPressLoginSection: Section {
+        let headerText = NSLocalizedString("WordPress.com login", comment: "Jetpack Settings: WordPress.com Login settings")
+        let footerText = NSLocalizedString("Learn more...", comment: "Discussion Settings: Footer Text")
+        let rows = [
+            Row(style: .Switch,
+                title: NSLocalizedString("Allow WordPress.com login", comment: "Jetpack Settings: Allow WordPress.com login"),
+                enabled: true,
+                boolValue: self.settings.jetpackSSOEnabled,
+                handler: { [weak self] in
+                    self?.pressedAllowWordPressComLogin($0)
+                }),
+
+            Row(style: .Switch,
+                title: NSLocalizedString("Match accounts using email", comment: "Jetpack Settings: Match accounts using email"),
+                enabled: self.settings.jetpackSSOEnabled,
+                boolValue: self.settings.jetpackSSOMatchAccountsByEmail,
+                handler: { [weak self] in
+                    self?.pressedMatchAccountsUsingEmail($0)
+                }),
+
+            Row(style: .Switch,
+                title: NSLocalizedString("Require two-step authentication", comment: "Jetpack Settings: Require two-step authentication"),
+                enabled: self.settings.jetpackSSOEnabled,
+                boolValue: self.settings.jetpackSSORequireTwoStepAuthentication,
+                handler: { [weak self] in
+                    self?.pressedRequireTwoStepAuthentication($0)
+                })
+        ]
+
+        return Section(headerText: headerText, footerText: footerText, rows: rows)
+    }
+
+    // MARK: - Private Nested Classes
+
+    fileprivate class Section {
```

Comment:
This seems to be using the same Section/Row system than discussion and notification settings. If we're going to have several copies of these, I'd rather start converging on using ImmuTable, or adopting a common system if ImmuTable is too static for the requirements.

Reply:
I'm changing it to ImmuTable, please hold on testing functionality until that's completed. Thx 👍

Reply:
and Done!